src/components/YouAppearToBeOffline.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withNetwork} from './OnyxProvider';
+import networkPropTypes from './networkPropTypes';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import variables from '../styles/variables';
+import Text from './Text';
+import styles from '../styles/styles';
+import compose from '../libs/compose';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+
+const propTypes = {
+    /** Information about the network */
+    network: networkPropTypes.isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const YouAppearToBeOffline = (props) => {
```

Comment:
There was consensus in the Design Doc to call this component `OfflineIndicator` because that describes what it is, not what it says or does.

Reply:
Hm, OK. I personally feel like that is not really a great name for it because an "offline indicator" could be done in several different ways (just an icon, just text, a spinner, something else, etc.). I do not feel too strongly about it at this point though, so I will change the name.

Reply:
That's a good point. I'm also not dug in on `OfflineIndicator` either, just going by the threads in the doc. I think @iwiznia was more strongly opinionated about it. Thanks for making the change anyway. Down to rename if we re-discuss and end up switching back.

Reply:
YouAppearToBeOffline also does not tell you if it is an icon, just text, a spinner, something else, no?